Classes/Finishers/EmailFinisher.php

```diff
@@ -96,7 +107,7 @@ protected function executeInternal()
             throw new FinisherException('The option "senderAddress" must be set for the EmailFinisher.', 1327060210);
         }
 
-        $mail = new \Neos\SwiftMailer\Message();
+        $mail = new SwiftMailerMessage();
```

Comment:
No, importing namespaces is just "syntactic sugar" that doesn't have an effect on the runtime.
I just tested it and without the `Neos.SwiftMailer` package installed I now get:

![image](https://user-images.githubusercontent.com/307571/29607516-c334d6f6-87f1-11e7-9841-02366a40bd9a.png)